docs/source/tutorials/text.ipynb

```diff
@@ -56,7 +106,7 @@
    "source": [
     "# Package installation (hidden on docs website).\n",
     "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
-    "\n",
+    "# Package versions we used: tensorflow==2.8.1 keras==2.8.0 numpy==1.21.6 pandas==1.3.2 scikeras==0.6.1\n",
     "dependencies = [\"cleanlab\", \"sklearn\", \"tensorflow\", \"tensorflow_datasets\", \"scikeras\"]\n",
```

Comment:
sorry not an issue introduced by your PR, but could we also get rid of the sklearn in the `dependencies` list from all the tutorials?

sklearn is already a dependency of cleanlab, so we should actually not be telling users to separately install it.

Reply:
Getting rid of sklearn, pandas, numpy and tqdm from here (which are all cleanlab dependencies in setup.py)